src/utils/L.Map.Sync.js

```diff
@@ -68,11 +68,9 @@ L.Map.include({
         const syncedMaps = this._syncedMaps[id]
 
         if (syncedMaps) {
-            for (let i = 0; i < syncedMaps.length; i++) {
-                if (syncedMaps[i] === this) {
-                    syncedMaps.splice(i, 1)
-                    i--
-                }
+            const idx = syncedMaps.indexOf(this)
+            if (idx !== -1) {
```

Comment:
This could also be `idx >= 0`, whichever you prefer though.

Reply:
Keeping -1